### PR TITLE
chore: add deprecation messages

### PR DESCRIPTION
**Pull request by lmiller1990**
Add deprecations as per https://github.com/dobromir-hristov/rfcs/blob/vtu-2-api/active-rfcs/0000-vtu-api.md

V1 is coming soon!

### CODE REVIEW

File: packages/test-utils/src/wrapper.js [hunk 1]
```diff
@@ -117,6 +117,9 @@ export default class Wrapper implements BaseWrapper {
    * Checks if wrapper contains provided selector.
    */
   contains(rawSelector: Selector): boolean {
+    warn(
+      'contains is deprecated and will be removed in a future release. Use `wrapper.find`, `wrapper.findComponent` or `wrapper.get`'
```
Comment (afontcu) [inline packages/test-utils/src/wrapper.js]:
```suggestion
      'contains is deprecated and will be removed in a future release. Use `wrapper.find`, `wrapper.findComponent` or `wrapper.get` instead'
```

File: packages/test-utils/src/wrapper.js [hunk 2]
```diff
@@ -258,6 +309,8 @@ export default class Wrapper implements BaseWrapper {
    * Checks if node is empty
    */
   isEmpty(): boolean {
+    warn(`isEmpty is deprecated and will be removed in a future release.
```
Comment (afontcu) [inline packages/test-utils/src/wrapper.js:312]: Nice 🙌

File: packages/test-utils/src/wrapper.js [hunk 3]
```diff
@@ -430,6 +486,9 @@ export default class Wrapper implements BaseWrapper {
    * Checks radio button or checkbox element
    */
   setChecked(checked: boolean = true): void {
+    warn(
+      `setChecked has been renamed setValue. setChecked will be removed in a future release.`
+    )
```
Comment (afontcu) [inline packages/test-utils/src/wrapper.js:491]: Should we warn about these? There's no alternative in VTU v1, and fixing this breaking change is going to be quite painless.

Comment (lmiller1990) [reply packages/test-utils/src/wrapper.js:491]: Maybe not. I'll remove them.

**Comment by lmiller1990** [comment]
closed in favor of https://github.com/vuejs/vue-test-utils/pull/1514

I messed up somehow and now it's conflicting idk why

**Pull request closed without merging**